frontend/src/layout/Burger.tsx

```diff
@@ -16,28 +16,52 @@ export const BurgerMenu: React.FC<BurgerMenuProps> = ({ hide, items }) => (
             bottom: 0,
             left: 0,
             right: 0,
-            zIndex: 3017,
+            zIndex: 1000,
             backgroundColor: "#000000a0",
         }}
     >
-        <div css={{
+        <div tabIndex={-1} css={{
             position: "absolute",
             display: "flex",
             flexDirection: "column",
             justifyContent: "space-between",
             top: 0,
             right: 0,
-            backgroundColor: "var(--grey97)",
+            backgroundColor: "white",
             height: "100%",
             width: "clamp(260px, 75%, 450px)",
             overflowY: "auto",
             borderTop: "1px solid var(--grey80)",
-            gap: 16,
-            "& > *:first-child": {
-                borderBottom: "1px dashed var(--grey80)",
+            "> :nth-child(2)": { marginTop: 26 },
```

Comment:
Realized that this needs to look at the second child rather than the last, as there is only one child when the user is logged out.